lib/c-api/Cargo.toml

```diff
@@ -39,6 +39,9 @@ paste = "0.1"
 # Commented out for now until we can find a solution to the exported function problem
 # wasmer-wasm-c-api = { version = "1.0.0-alpha5", path = "crates/wasm-c-api" }
 
+[dev-dependencies]
+inline-c = { git = "https://github.com/Hywan/inline-c-rs", branch = "fix-linux-shared-object" }
```

Comment:
Can we use a published version of the crate?

Reply:
Yes we will. We are waiting for the last bits of https://github.com/Hywan/inline-c-rs/pull/4 before releasing the crate.